src/metrics.ts

```diff
@@ -280,4 +294,17 @@ export default class Metrics {
       payIdCount,
     )
   }
+
+  private updateInfoGauge(): void {
```

Comment:
Do we need to add a TS/JSDoc to this function?

Reply:
Documented.